AGR-469

pjhale opened the pull request:
Fix for Jira issue 469.  Added strand and build from index.
Changed heading from Gene Model to Sequence Feature Viewer.
Removed button and created a link with the location and build information
above the sequence viewer.
@nathandunn @paaatrick

pjhale committed:
AGR-469

Fix for Jira issue 469.  Added strand and build from index page.
Changed heading from Gene Model to Sequence Feature Viewer.
Removed button and created a link with the location and build information
above the sequence viewer.

nathandunn commented:
@pjhale  I think it looks good.  Minor requests:

- Match the section above (see "Additional Info").   So: 
  - no link on Genome Location
  - bold and flush left for Genome Location
  

<img width="593" alt="screen shot 2017-07-28 at 10 52 44 am" src="https://user-images.githubusercontent.com/751274/28729908-fa5d3c1e-7382-11e7-96ee-04300c18304a.png">

pjhale committed:
Updated formatting to match

Updated to have the formatting the same as the rest of the
gene page.  Also removed the link from the Genome Location text.

pjhale commented:
Updated formatting in 2nd commit.

nathandunn reviewed (approved):
Looks good to me.   I'm wondering if @paaatrick  wants to institute a standard header bar across all of the elements, though.  Its done in half of them (and all a bit differently).   Anyway, beyond this scope.
